docs/development/bump-node-major.md

```diff
@@ -0,0 +1,18 @@
+# How to bump Renovate to next NodeJS LTS release
+
+## Add new NodeJS version
+
+- Add new versions via `package.json>engines>node` and `package.json>engines-next>node`
```

Comment:
```suggestion
- Add new version via `package.json>engines>node` and `package.json>engines-next>node`
```

Whoops, missed a spot.
We're using the singular form `version` everywhere else, so probably should be consistent throughout.